### PR TITLE
Placement: reduce verbosity of a debug log

### PR DESCRIPTION
**Pull request by ItalyPaleAle**
When running placement with debug logs, line `Process Raft state command: nothing happened...` is getting logged every 10s, causing a lot of unnecessary noise that makes debugging harder.

### CODE REVIEW

File: pkg/placement/membership.go [hunk 1]
```diff
@@ -264,7 +264,7 @@ func (p *Service) processRaftStateCommand(ctx context.Context) {
 			return
 
 		case <-processingTicker.C():
-			log.Debugf("Process Raft state command: nothing happened...")
+			// log.Debugf("Process Raft state command: nothing happened...")
```
Comment (JoshVanL) [inline pkg/placement/membership.go:267]: I agree with the change though I think this line should be deleted completely, or replaced with a proper code comment.

**Review by JoshVanL — CHANGES_REQUESTED** [review]

File: pkg/placement/membership.go (hunk 1 above)
Comment (ItalyPaleAle) [reply pkg/placement/membership.go:267]: Thinking about it, i don't think we even need the `processingTicker`. Its sole purpose was to show this log...

Comment (ItalyPaleAle) [reply pkg/placement/membership.go]: Removed the ticker entirely

**Review by JoshVanL — APPROVED** [review]

**Review by mukundansundar — APPROVED** [review]
we had a ticker only for logging?

**Comment by ItalyPaleAle** [comment]
> we had a ticker only for logging?

Looks like that was the case, yes

**Pull request merged**